packages/gatsby-theme-sso-ui-kit/src/@commercetools-docs/gatsby-theme-docs/overrides/avatar.js

```diff
@@ -1,179 +1,110 @@
 // A React component to be rendered in the top bar next to the top menu toggle button
-import { useEffect, useState, useContext } from 'react';
+import { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import md5 from 'md5';
-import { useSelect } from 'downshift';
```

Comment:
I think `downshift` dependency can be removed at all from `package.json`

Reply:
Right! Good catch 👍🏼